lib/prawn/vera_pdf.rb

```diff
@@ -0,0 +1,55 @@
+require 'nokogiri'
+require 'open3'
+
+module Prawn
+  module VeraPdf
```

Comment:
You don't need to comment on each my comment as long as you push granular commits. GitHub hides comments on the code that has been changed and lets reviewing only new changes. It also sends emails about new commits (not about force pushes, unfortunately, so please let me know about those if you want someone to look at those).

Just a hint to save a few minutes for you.